microsite/sidebars.json

```diff
@@ -248,6 +248,15 @@
       "plugins/internationalization",
       "plugins/analytics",
       "plugins/feature-flags",
+      {
+        "type": "category",
+        "label": "OpenAPI",
+        "items": [
+          "openapi/01-getting-started",
+          "openapi/generate-client",
+          "openapi/test-case-validation"
+        ]
+      },
```

Comment:
I believe you additionally have to add the category to the `mkdocs.yml` - like e.g. for "Backends and APIs" (https://github.com/backstage/backstage/blob/master/mkdocs.yml#L131-L135)

Missed it also before - bit confusing - hope that helps! :)

Reply:
Updated! Is there an easy way to test that locally? I was getting weird yarn errors with the existing `techdocs-cli` command.

Reply:
I would expect that you can test it by running the local dev of the microsite. Not sure about the `techdocs-cli` errors :/